frontend/app/src/App.vue

```diff
@@ -186,6 +241,15 @@ export default class App extends Vue {
   display: block;
   width: 100%;
   content: '';
-  border-bottom: #bcbcbc solid thin;
+  border-bottom: var(--v-rotkiLightGrey-darken1) solid thin;
+}
+
+.app-bar__button ::v-deep i:focus,
+.app-bar__button ::v-deep button:focus {
+  color: var(--v-primary-base) !important;
+}
+
+::v-deep .v-content__wrap {
```

Comment:
is this class valid after vuetify 2.3.x where v-content was renamed to v-main?

Reply:
Will check

Reply:
Fixed to 'v-main'